deepmd/pt/utils/dataset.py

```diff
@@ -694,13 +532,8 @@ def single_preprocess(self, batch, sid):
                             f"Add noise times beyond max tries {self.max_fail_num}!"
                         )
                     continue
-                batch["atype"] = masked_type
+                batch["type"] = masked_type
                 batch["coord"] = noised_coord
```

Comment:
My understanding is that this part needs to be removed completely, and the placement of the addition of noise needs to be discussed later.

Reply:
This interface and process code may leave for you to clean up in an individual PR.